app/components/task_list/view.rb

```diff
@@ -0,0 +1,43 @@
+class TaskList::View < GovukComponent::Base
+  include ViewComponent::Slotable
+
+  with_slot :row, collection: true, class_name: "Row"
+  wrap_slot :row
```

Comment:
Interesting. Did some looking into this, and this whole `wrap_slot` is giving me the chills. It [redefines `#row` with a method that just calls `component.slot(:row, ...)`](https://github.com/DFE-Digital/govuk-components/blob/0a9fb7391674b977081be1c912268716415edac4/app/components/govuk_component/base.rb#L10-L16). This overrides [the accessor that was defined by ViewComponent::Slotable with](https://github.com/github/view_component/blob/828714eaaeb62fbca8928ce588725d3ca2e7a2ff/lib/view_component/slotable.rb#L44-L53), as above, something that calls `component.slot(:row, ...)`. That method, unlike the accessor that was just wiped out, [returns nil](https://github.com/github/view_component/blob/828714eaaeb62fbca8928ce588725d3ca2e7a2ff/lib/view_component/slotable.rb#L118).

TL;DR: `wrap_slot` as-is breaks the `row` accessor (this issues is discussed here: https://github.com/github/view_component/discussions/454#discussioncomment-79355)

I'll make a comment on `govuk-components`.

Reply:
https://github.com/DFE-Digital/govuk-components/issues/63

Reply:
Seeing as the "wrapped" slot isn't used below, I'd suggest removing the `wrap_slot` for now.

Reply:
I've removed that line now